assets/src/components/mapPage/mapDisplay.tsx

```diff
@@ -361,21 +362,22 @@ const SelectedVehicleDataLayers = ({
               />
             </SelectionCardContainer>
           )}
-          {isVehicle(selectedVehicleOrGhost) &&
-            (selectedVehicleOrGhost?.isShuttle ? (
+          {isVehicle(selectedVehicleOrGhost) && (
+            <>
               <VehicleMarker
                 key={selectedVehicleOrGhost.id}
                 vehicle={selectedVehicleOrGhost}
                 isPrimary={true}
                 isSelected={true}
               />
-            ) : (
+
               <RouteVehicles
```

Comment:
question: Is it intentional for `<RouteVehicles/>` to be included for shuttles?

Reply:
Not necessarily, I guess I need to think this part of the logic over a little bit more carefully.